content/08-mongodb/01-what-is-mongodb.mdx

```diff
@@ -0,0 +1,109 @@
+
+</PrismaOutlinks>
+
+## Origin story
+
+MongoDB was founded in 2007 by Dwight Merriman, Eliot Horowitz, and Kevin Ryan. They previously worked at an internet advertising company that was serving  400,000 ads per second. 
+
+The team was developing many custom data stores to accommodate this traffic. They regularly ran into struggles around scalability and agility, which led to the inspiration to create a database that solved for these issues. And voila, MongoDB. 
+
+## How it works
+To achieve better scalability and agility, MongoDB works with the JSON variant, Binary JSON, or [BSON](https://www.prisma.io/dataguide/mongodb/mongodb-datatypes).  BSON accommodates more data types than JSON giving more flexibility to the types of data that can be stored in MongoDB.
+
+<PrismaOutlinks>
+
+If you are looking to get started working with MongoDB and Prisma, checkout our [getting started from scratch guide](https://www.prisma.io/docs/getting-started) or how to [add to an existing project](https://www.prisma.io/docs/getting-started/setup-prisma/add-to-existing-project/mongodb-typescript-mongodb).
+
+</PrismaOutlinks>
+
+### MongoDB data Structure
+Rather than tables made up of rows and columns, MongoDB has collections made up of documents. Documents are comprised of field and value pairs. In relation to relational databases, the pieces that make up a MongoDB database can loosely be thought of as follows:
+
+| Relational |   | Document |
+|----------- |---|--------- |
+|Table       | = |Collection|
+|Row | = | Document |
+|Column | =| Field|
+
+The basic structure looks as follows:
+
+```
+{
+   field1: value1,
+   field2: value2,
+   field3: value3,
+   ...
+   fieldN: valueN
+}
+```
+
+Documents look very similar to a JSON object. Documents can hold any of the available BSON data types as well as other documents, arrays, and arrays of documents. This JSON-style format directly maps to native objects in most modern programming languages, making it a natural choice for developers.
+
+ An example would look like:
+
+```
+var mydoc = {
+               _id: ObjectId("5099803df3f4948bd2f98391"),
+               name: { first: "James", last: "Joyce" },
+               birth: new Date('Feb 02, 1882'),
+               death: new Date('Jan 14, 1941'),
+               books: [ "Ulysses", "Dubliners","Finnegan's Wake" ],
+               sales : NumberLong(1250000)
+            }
+```
+
+- `_id` holds an [*ObjectId*].
+- `name` holds an *embedded document* that contains the fields `first` and `last`.
+- `birth` and `death` hold values of the [*Date*] type.
+- `books` holds an array of [*strings*].
+- `sales` holds a value of the [*NumberLong*] type.
+
+### MongoDB query language
+Since MongoDB databases are document oriented, the data is typically unstructured. This makes accessing data different then when querying a relational database with structured query language (SQL). MongoDB has its own query language simply called MongoDB Query Language (MQL). 
```

Comment:
I always wonder with MongoDB if there's a better way of saying that the data is unstructured.  Is it more accurate to say that it doesn't follow a predetermined structure or something similar?  I think it's just a weird thing that my brain gets stuck on that I'm sure other people don't worry about.